src/test/java/parser/ParserTest.java

```diff
@@ -0,0 +1,31 @@
+package parser;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class ParserTest {
+
+    @Test
+    public void isMissingDeadlineInformationTest() {
```

Comment:
I like how thorough you are with test cases! However I think naming convention should be something like: 
isMissingDeadlineInformation_correctFormat_success() or isMissingDeadlineInformation_incorrectFormat_exceptionThrown() 
to show unitBeingTested_descriptionOfTestInputs_expectedOutcome in the test name

Reply:
Thank you for the feedback:)